pkg/internal/token/testdata/ROPCPoPTokenFromBadPasswordVCR.yaml

```diff
@@ -0,0 +1,247 @@
+        headers:
+            Access-Control-Allow-Methods:
+                - GET, OPTIONS
+            Access-Control-Allow-Origin:
+                - '*'
+            Cache-Control:
+                - max-age=86400, private
+            Client-Request-Id:
+                - e970f474-9424-489a-a3ec-7fa084dccbfc
+            Content-Length:
+                - "1753"
+            Content-Type:
+                - application/json; charset=utf-8
+            Date:
+                - Thu, 15 Feb 2024 20:34:43 GMT
+            P3p:
+                - CP="DSP CUR OTPi IND OTRi ONL FIN"
+            Strict-Transport-Security:
+                - max-age=31536000; includeSubDomains
+            X-Content-Type-Options:
+                - nosniff
+            X-Ms-Ests-Server:
+                - 2.1.17338.4 - NCUS ProdSlices
+            X-Xss-Protection:
+                - "0"
+        status: 200 OK
+        code: 200
+        duration: 303.396052ms
+    - id: 1
+      request:
+        proto: ""
+        proto_major: 0
+        proto_minor: 0
+        content_length: 0
+        transfer_encoding: []
+        trailer: {}
+        host: ""
+        remote_addr: ""
+        request_uri: ""
+        body: ""
+        form: {}
+        headers:
+            Accept-Encoding:
+                - gzip
+            Client-Request-Id:
+                - 5af9b9df-8ad8-445d-80e6-f767b81a8ef7
+            Return-Client-Request-Id:
+                - "false"
+            X-Client-Cpu:
+                - amd64
+            X-Client-Os:
+                - linux
+            X-Client-Sku:
+                - MSAL.Go
+            X-Client-Ver:
+                - 1.2.0
+            client-request-id:
+                - 127294a4-5ca2-4276-a3df-d3fb175a405e
+        url: https://login.microsoftonline.com/common/UserRealm/k8connectsa@microsoft.com?api-version=1.0
```

Comment:
`k8connectsa@microsoft.com` this needs to be fuzzed

Reply:
Redacted the username